Rules/CommonScripts/ChatCommands/MaterialCommands.as

```diff
@@ -0,0 +1,109 @@
+#include "ChatCommand.as"
+
+class AllMatsCommand : ChatCommand
+{
+	AllMatsCommand()
+	{
+		super("allmats", "Spawn all types of materials");
+		AddAlias("allmaterials");
+		AddAlias("materials");
+		AddAlias("mats");
+	}
+
+	void Execute(string[] args, CPlayer@ player)
+	{
+		if (!isServer()) return;
+
+		CBlob@ blob = player.getBlob();
+		if (blob !is null)
+		{
+			Vec2f pos = blob.getPosition();
```

Comment:
Also affected by the blob spawning comment (posting a comment for each of them for tracking)